src/main/kotlin/org/jdc/template/datasource/database/main/individual/Individual.kt

```diff
@@ -21,6 +21,9 @@ class Individual {
     var phone: String = ""
     var email: String = ""
     var available: Boolean = false
+    var profileUrl: String = ""
 
     fun getFullName() = "$firstName $lastName"
+
+    fun getFullData() = "$id $householdId $individualType $firstName $lastName $birthDate $alarmTime $lastModified $phone $email $available $profileUrl"
```

Comment:
You could convert the class to a data class (see https://kotlinlang.org/docs/reference/data-classes.html) then this would not be needed.